scripts/website.js

```diff
@@ -10,11 +10,16 @@ const pug = require('pug');
 const pkg = require('../package.json');
 const transform = require('acquit-require');
 const childProcess = require("child_process");
-require('./generateSearch');
+
 // using "__dirname" and ".." to have a consistent CWD, this script should not be runnable, even when not being in the root of the project
 // also a consistent root path so that it is easy to change later when the script should be moved
 const cwd = path.resolve(__dirname, '..');
 
+// support github-like custom header ids
+// Example:
+// # Some Header {#custom-id}
+const CustomIdRefex = /{#([a-z][\w-]*)}(?: *)$/;
```

Comment:
```suggestion
const CustomIdRefex = /{#([a-zA-Z0-9_-]+)}(?: *)$/;
```

add `_` as otherwise `guide.md`'s `option: _id` does not get recognized, also expand `\w` to be more explicit in what is supported and only have one range (as we are in control of both processing and writing the headings)